### PR TITLE
set_device after enable_static & 2d -> 2D & --dynamic -> --static

### PR DESCRIPTION
**Pull request by heavengate**
**set_device after enable_static**

### CODE REVIEW

File: tsm/infer.py [hunk 1]
```diff
@@ -33,8 +33,8 @@
 
 
 def main():
+    paddle.enable_static(device) if not FLAGS.dynamic else None
```
Comment (qingqing01) [inline tsm/infer.py:36]: there is no device before line 36

File: yolov3/infer.py [hunk 2]
```diff
@@ -62,8 +62,8 @@ def load_labels(label_list, with_background=True):
 
 
 def main():
+    paddle.enable_static(device) if not FLAGS.dynamic else None
```
Comment (qingqing01) [inline yolov3/infer.py:65]: same as above

File: tsm/infer.py (hunk 1 above)
Comment (heavengate) [reply tsm/infer.py]: Done, thanks!

File: yolov3/infer.py (hunk 2 above)
Comment (heavengate) [reply yolov3/infer.py]: Done, thanks!

**Review by LielinJiang — APPROVED** [review]

**Pull request merged**